### PR TITLE
Add user to org

### PR DESCRIPTION
**Pull request by smurfik**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -27,6 +27,12 @@ def create
     @user = User.new(params.require(:user).permit(:email, :first_name, :last_name, :password, :password_confirmation))
     if @user.save
       session[:user_id] = @user.id
+      @org_invites = OrganizationInvite.where("email = ? and created_at >= ?", @user.email, 1.week.ago)
+      if @org_invites.length > 0
```
Comment (bookis) [inline app/controllers/users_controller.rb]: I believe you can do `if @org_invites.exists?` which will do a SQL command which checks if any exist, which may be more efficient (and it reads a bit better).

File: app/controllers/organization_invites_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class OrganizationInvitesController < ApplicationController
+  def new
+    @organization = Organization.find(params[:organization_id])
+  end
+
+  def create
```
Comment (bookis) [inline app/controllers/organization_invites_controller.rb:7]: I know we already talked about this, but this one will be a great one to refactor.

**Comment by smurfik** [comment]
made the changes --- can't figure out how to refactor create method further -- any suggestions?

**Comment by bookis** [comment]
Yeah, but probably easier to talk the strategy through. Do you want to walk through it next time you meet up with us?

**Comment by smurfik** [comment]
sounds good

**Comment by smurfik** [comment]
just made the changes that we talked about

**Pull request merged**